packages/yoga/src/Result/native/Result/index.jsx

```diff
@@ -95,6 +71,8 @@ Result.propTypes = {
   children: node,
   /** The color of attendences icon and description */
   attendancesColor: string,
+  /** The property that defines a partner as exclusive */
+  exclusivity: bool,
```

Comment:
❓ We should always try to make the components the more agnostic as possible. In this case, wouldn't make more sense to have a `badge` prop with an `Icon` as value?

That way, we could use any Icon as a badge.
```suggestion
  /** The property that defines a whether a badge should be shown and which icon should be rendered */
  badge: node,
```